tests/unit/mainwindow/test_prompt.py

```diff
@@ -130,3 +131,14 @@ def test_root_path(self, get_prompt):
         """With / as path, show root contents."""
         prompt = get_prompt('/')
         assert prompt._file_model.rootPath() == '/'
+    
+    def test_expand_user(self, qtbot, get_prompt):
+        prompt = get_prompt('/')
+        prompt._lineedit.setText('')
+        qtbot.keyPress(prompt._lineedit, Qt.Key_AsciiTilde)
```

Comment:
Does this need to be a keypress, or could you use `setText('~')` above?

Reply:
Darn it I thought that was what I did. I was considering doing both and I thought I had set the text to ~.

Reply:
I remember why I did it. For whatever reason setting the text doesn't seem to call the `_set_fileview_root` function and if I am pretty sure if it did it would cause a loop. I remember looking into this when writing the test for the fuzzyfinding.

Reply:
Could you please add a comment explaining that?